src/components/Simulation/index.js

```diff
@@ -53,23 +62,20 @@ export const Simulation = props => {
   const [packages, setPackages] = useState([]);
   const [orgUnits, setOrgUnits] = useState([]);
 
-  const { errorMessage, loading, payload, history, simulationData, t } = props;
+  const { errorMessage, loading, invoices, request, history, t, open } = props;
 
-  const simulations = payload.invoices;
   const isLoaded = !loading;
   const hasError = !!errorMessage;
   const isSuccess = isLoaded && !hasError;
-  const open = simulationData && !!simulationData.id;
-
-  const name = simulationData && simulationData.name;
-  const formattedDate = simulationData && simulationData.period;
-  const nameWithDate = `${name}-${formattedDate}`;
+  const nameWithDate = !request
+    ? "…"
```

Comment:
Can we make this into a `&hellip;`?

Reply:
Any reason? «…» is valid utf8 character :)

Reply:
I think old age. In UTF8 it probably won't matter if we actually use the HTML entity or not, but using a html entity is usually easier to read. So to make it really clear we mean the &hellip; and not just three dots, I prefer the &hellip; (not a strong opinion, so feel free to pick whichever one you like more).